GitUI/Editor/FileViewerInternal.cs

```diff
@@ -712,7 +719,8 @@ public int CurrentFileLine(bool isDiff)
                         : viewPosition.ActiveLineNum.RightLineNumber;
                 }
 
-                return viewPosition.CaretPosition.Line;
+                // Convert from offset to line number
+                return viewPosition.CaretPosition.Line + 1;
```

Comment:
The reported file number was incorrect (so wrong line was opened in Blame, where this was used).